android/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
         android:icon="@mipmap/ic_launcher">
         <activity
             android:name=".MainActivity"
-            android:launchMode="singleTop"
+            android:launchMode="singleTask"
```

Comment:
why are we changing the launchMode ?

Reply:
Before, whenever we open the atmosphere pro by sharing from gallery, the app opens up in a new instance every time. So there were multiple instances of the app and the sharing was not working.  

So this fixes the instance problem and hence sharing intent works on android